scripts/bundle.ts

```diff
@@ -57,7 +57,7 @@ module.exports.default = react;`,
   copyFileSync("README.md", "dist/README.md");
 
   execSync(
-    "tsc src/index.ts --declaration --emitDeclarationOnly --outDir dist --module es2020 --moduleResolution bundler",
+    "tsc src/index.ts --declaration --emitDeclarationOnly --outDir dist --module es2020 --moduleResolution bundler --skipLibCheck",
```

Comment:
`skipLibCheck` is always recommended, so I'd suggest to keep it anyway, even if the cause fixed in `node_modules`

Reply:
Yeah I recommend it for apps because this its; often not actionnable but here it allows me to catch regressions in Vite shipped by the the lib I used and that I can send a PR too (it helped fixed bad types in Vite or LightningCSS in the past)

Reply:
ok